index.js

```diff
@@ -106,6 +106,14 @@ async function register (server, options) {
 
   // log when a request completes
   tryAddEvent(server, options, 'on', 'response', function (request) {
+    if (options.ignorePaths && ignoreTable[request.url.path]) {
+      return
+    }
+
+    if (!request.response) {
+      return
+    }
```

Comment:
Why is this check needed? code coverage is reporting that this is not covered by tests: https://coveralls.io/builds/18603822/source?filename=index.js.